hack/toolchain.sh

```diff
@@ -17,12 +14,11 @@ tools() {
 }
 
 kubebuilder() {
-    os=$(go env GOOS)
-    arch=$(go env GOARCH)
-    curl -L "https://go.kubebuilder.io/dl/2.3.1/${os}/${arch}" | tar -xz -C $TEMP_DIR
-    sudo mkdir -p /usr/local/kubebuilder/bin/
-    sudo mv $TEMP_DIR/kubebuilder_2.3.1_${os}_${arch}/bin/* /usr/local/kubebuilder/bin/
-    echo 'Add kubebuilder to your path `export PATH=$PATH:/usr/local/kubebuilder/bin/`'
+    KUBEBUILDER_ASSETS="/usr/local/kubebuilder"
```

Comment:
It's unfortunately how the mv command works. I need to clear the previous folder to rename the new folder into place. Otherwise the files get copied into the existing folder.